kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala

```diff
@@ -181,36 +181,51 @@ class KyuubiRestFrontendService(override val serverable: Serverable)
   @VisibleForTesting
   private[kyuubi] def recoverBatchSessions(): Unit = withBatchRecoveryLockRequired {
     val recoveryNumThreads = conf.get(METADATA_RECOVERY_THREADS)
+    val recoveryBatchSize: Int = conf.get(BATCH_SESSIONS_RECOVERY_SIZE)
```

Comment:
How about directly use `recoveryNumThreads` as the batch size? Then we can avoid import a new config here, and if the batch size larger than `recoveryNumThreads`, seems meaningless.

Reply:
If we want greater concurrency, we should be able to increase this value. 
My concern is that we've added an extra configuration parameter, but this parameter is actually only used to control the batch size for queries from the database. Query results are also subject to `recoveryNumThreads ` here.
If we use `recoveryNumThreads` as the batch size, it would be simpler to use, we only need to consider concurrency.

Reply:
OK, I'm not sure if I understand this scene. In my view, what needs to be done here is to query the tasks requiring recovery from the database and then commit them. If the number of tasks queried exceeds the number of concurrent commits, it seems pointless.